packages/tasks/src/model-libraries.ts

```diff
@@ -553,6 +553,14 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: true,
 		countDownloads: `path_extension:"sentis"`,
 	},
+	"videocrafter": {
+		prettyLabel: "VideoCrafter",
```

Comment:
Do we want to distinguish VideoCrafter and VideoCrafterV2? Simply asking 'cause I don't know the difference (if any)

Reply:
I think it's fine as both are in the same Github repository and have the same structure